cmd/juju/resource/deploy_test.go

```diff
@@ -322,6 +325,222 @@ func (s DeploySuite) TestMissingResource(c *gc.C) {
 	c.Check(errors.Cause(err), jc.Satisfies, os.IsNotExist)
 }
 
+func (s DeploySuite) TestDeployDockerResourceRegPathString(c *gc.C) {
```

Comment:
This test seems to be broken on windows tests. http://ci.jujucharms.com/job/unit-tests-win2012/487/console